composables/config.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { defu } from 'defu'
+import type { CustomAppConfig } from '../.nuxt/schema/nuxt.schema'
+
+type ToolkitConfig = NonNullable<CustomAppConfig['exactproDocs']>
+
+export function useToolkitConfig(): ToolkitConfig {
+  const appConfig = useAppConfig()
+  const docsConfig = appConfig.exactproDocs
+  const computedDefaultValues: ToolkitConfig = {
+    seo: {
+      sitemap: {
+        baseUrl: docsConfig.seo?.baseUrl
```

Comment:
![image](https://github.com/exactpro/docs-toolkit/assets/102023206/9f19467b-10a5-433e-aa5c-ba800eeed135)

Reply:
ran ```run npm prepare``` and this was resolved